lib/bundler/settings.rb

```diff
@@ -86,6 +105,7 @@ def gem_mirrors
     def locations(key)
       key = key_for(key)
       locations = {}
+      location[:current] = @current_config[key] if @current_config.key?(key)
```

Comment:
Ugh thanks for picking that up. As my friend told me: "You really wouldn't believe English is your first language."